apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/success/page.tsx

```diff
@@ -14,30 +14,41 @@ import {
   recordEmitterEventAction,
 } from '@fxa/payments/ui/actions';
 import { CheckoutParams } from '@fxa/payments/ui/server';
+import Image from 'next/image';
+import Amex from '@fxa/shared/assets/images/payment-methods/amex.svg';
+import Diners from '@fxa/shared/assets/images/payment-methods/diners.svg';
+import Discover from '@fxa/shared/assets/images/payment-methods/discover.svg';
+import Jcb from '@fxa/shared/assets/images/payment-methods/jcb.svg';
+import Mastercard from '@fxa/shared/assets/images/payment-methods/mastercard.svg';
+import Paypal from '@fxa/shared/assets/images/payment-methods/paypal.svg';
+import Unbranded from '@fxa/shared/assets/images/payment-methods/unbranded.svg';
+import UnionPay from '@fxa/shared/assets/images/payment-methods/unionpay.svg';
+import Visa from '@fxa/shared/assets/images/payment-methods/visa.svg';
 
 export const dynamic = 'force-dynamic';
 
-type ConfirmationDetailProps = {
-  title: string;
-  detail1: string | Promise<string>;
-  detail2: string;
-};
-
-const ConfirmationDetail = ({
```

Comment:
Ended up removing this component and moving it inline into the page jsx.

The 2nd usage of this component needed an image in detail 2, and instead of updating the component to support another compoent, I figured it'd be cleaner just to move the logic into the page JSX